test/background/stat_estimators.jl

```diff
@@ -0,0 +1,10 @@
+using Photometry.Background: sigma_clip
+
+@testset "sigma clipping" begin
+    x = [1, 2, 3]
+    @test sigma_clip(x, 1, 1) ≈ [1.0, 2.0, 3.0] rtol=1e-4
+    @test sigma_clip(x, 1) ≈ [1.0, 2.0, 3.0] rtol=1e-4
```

Comment:
I know this will seem redundant, but I recommend changing to this:
```julia
@test sigma_clip(x, 1, 1) ≈ [1.0, 2.0, 3.0] rtol=1e-4
@test sigma_clip(x, 1, 1) == sigma_clip(x, 1)
```

The first test is an acccuracy test, the second test tests the regression for sigma_high=sigma_low.